app/cdap/components/PipelineConfigurations/ConfigurationsContent/ResourcesTabContent/DriverResources.js

```diff
@@ -59,10 +59,12 @@ const mapDispatchToProps = (dispatch) => {
         type: PipelineConfigurationsActions.SET_DRIVER_VIRTUAL_CORES,
         payload: { virtualCores: e.target.value },
       });
-      dispatch({
-        type: PipelineConfigurationsActions.SET_RUNTIME_ARGS,
-        payload: { runtimeArgs: { pairs: newRunTimePairs } },
-      });
+      if (lifecycleManagementEditEnabled) {
```

Comment:
A little context would help here. Looks like we're setting runtime args only when `lifecycleManagementEditEnabled` is true? what happens to the usual scenario without lifecycle management?